packages/vapor/resources/icons/svg/unlock-stroked-large.svg

```diff
@@ -0,0 +1,5 @@
+<svg width="32" height="32" viewBox="0 0 32 32" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
We need to remove `width` and `height` attributes from the `svg` root in order to customize the icon size with classes. for example `mod-14`.